### PR TITLE
[DOR-39][AO] Landing page for the Route 1 check on Trader Services

### PR DESCRIPTION
**Pull request by arturopala**

### CODE REVIEW

File: conf/messages.en [hunk 1]
```diff
@@ -140,3 +137,21 @@ researchBanner.heading = Help improve GOV.UK
 researchBanner.inviteLink = Help improve digital services by joining the HMRC user panel (opens in a new window)
 researchBanner.closeLink = No thanks
 
+# Landing Page
+# ----------------------------------------------------------
+view.start.title=Trader services
```
Comment (Tasmine10ten) [inline conf/messages.en:142]: Proto has this has "Use trader services" but maybe Trader services is more appropriate?

File: conf/messages.en [hunk 2]
```diff
@@ -140,3 +137,21 @@ researchBanner.heading = Help improve GOV.UK
...
+view.start.heading=Trader services
+view.start.route-one=Upload documents for pre-clearance checks (routes 1, 2, 3, 6 )
+view.start.route-one.bullets.1=Upload documents for new,hold and cancellation consignment
+view.start.route-one.bullets.2=Upload documents for exports C1602, C1602, C1603
+view.start.route-one.bullets.3=Amend an exiting submission - add more files or change details
+view.start.route-one.link.text=Start a route 1 pre-clearance check submission
+view.start.reclaim=Reclaim duty you've overpaid
```
Comment (Tasmine10ten) [inline conf/messages.en:149]: Will need to be a smart apostrophe

**Review by Tasmine10ten — CHANGES_REQUESTED** [review]

File: app/uk/gov/hmrc/traderservices/views/StartView.scala.html [hunk 3]
```diff
@@ -30,5 +27,49 @@
...
+    <br><br>
+
+    @html.h2(messages("view.start.reclaim"))
+
+    @html.p{
+        @html.p{
+            @messages("view.start.reclaim.description")
+        }
+        @html.a("/trader-services/reclaim", messages("view.start.reclaim.link.text"))
+    }
```
Comment (slashrealm) [inline app/uk/gov/hmrc/traderservices/views/StartView.scala.html:51]: I can't find this page inside the app, so I can't be sure what html code this generates, but these look like nested P tags, which is incorrect html. There are few more instances below.

File: app/uk/gov/hmrc/traderservices/views/StartView.scala.html [hunk 4]
```diff
@@ -30,5 +27,49 @@
...
+
+    @html.ul(
+        messages("view.start.route-one.bullets.1"),
+        messages("view.start.route-one.bullets.2"),
+        messages("view.start.route-one.bullets.3")
+    )
+
+    @html.a("/trader-services/route-one", messages("view.start.route-one.link.text"))
+
+    <br><br>
```
Comment (slashrealm) [inline app/uk/gov/hmrc/traderservices/views/StartView.scala.html:42]: &lt;br&gt;s shouldn't be used for spacing.

File: app/uk/gov/hmrc/traderservices/views/StartView.scala.html [hunk 5]
```diff
@@ -30,5 +27,49 @@
...
+    @html.h2(messages("view.start.reclaim"))
+
+    @html.p{
+        @html.p{
+            @messages("view.start.reclaim.description")
+        }
+        @html.a("/trader-services/reclaim", messages("view.start.reclaim.link.text"))
+    }
+
+    <br>
```
Comment (slashrealm) [inline app/uk/gov/hmrc/traderservices/views/StartView.scala.html:53]: This &lt;br&gt; seems to be redundant (there's one more further down)

**Review by slashrealm — CHANGES_REQUESTED** [review]

**Comment by arturopala** [comment]
minor changes applied

File: app/uk/gov/hmrc/traderservices/views/StartView.scala.html (hunk 4 above)
Comment (arturopala) [reply app/uk/gov/hmrc/traderservices/views/StartView.scala.html:42]: I will leave to you as a frontend developer to propose something instead or remove. Feel free to amend the PR.

**Review by russellthorn — APPROVED** [review]
LGTM 👍

**Review by slashrealm — APPROVED** [review]

**Review by Tasmine10ten — APPROVED** [review]

**Review by slashrealm — APPROVED** [review]

**Pull request closed without merging**